portfolio/portfolio.go

```diff
@@ -472,70 +507,81 @@ func (b *Base) GetPortfolioGroupedCoin() map[currency.Code][]string {
 	return result
 }
 
-// Seed appends a portfolio base object with another base portfolio
-// addresses
-func (b *Base) Seed(port Base) {
-	b.Addresses = port.Addresses
-}
-
 // StartPortfolioWatcher observes the portfolio object
-func (b *Base) StartPortfolioWatcher() {
-	addrCount := len(b.Addresses)
-	log.Debugf(log.PortfolioMgr,
-		"PortfolioWatcher started: Have %d entries in portfolio.\n", addrCount,
-	)
+func (b *Base) StartPortfolioWatcher(ctx context.Context, sleepDelay time.Duration) error {
```

Comment:
🐒 It's not really a sleep delay, it's an interval / `checkInterval` / `runInterval`.
I would have expected a sleepDelay to mean that the watcher will wait Duration before starting operations.

```suggestion
func (b *Base) StartPortfolioWatcher(ctx context.Context, interval time.Duration) error {
```